SECURITY.md

```diff
@@ -1,7 +1,11 @@
-# Security Policy
+# Security Policy 
+ 
+## Security Contacts and Procedures
+ 
+This community takes security very seriously, and we aim to take immediate action to address serious security-related problems that involve our products or services.
+ 
+Please report any suspected security vulnerability in this project to Red Hat Product Security at secalert@redhat.com. You can use our GPG key to communicate with us securely.
+ 
+To report an issue in any Red Hat branded website or online service, please contact Red Hat Information Security at site-security@redhat.com.
+https://access.redhat.com/security/team/contact
 
-## Reporting a Vulnerability
-
-Any security vulnerability can be reported at the project [JIRA](https://issues.redhat.com/projects/WELD).
```

Comment:
@luck3y is there any specific issue with keeping the JIRA approach?
I know sec issue belong to secalert, but this way it will go primarily there and I will only learn of them by proxy with delay.
Personally, I think the flow where I learn of the issue first and then contact secalert if needed makes more sense.
Or is there something I am missing?